ansible/roles/base/tasks/main.yml

```diff
@@ -1,18 +1,41 @@
 
 ---
 
+    - name: Check NTP service to make sure its running
+      service: name=ntp state=started enabled=yes
+      become: yes
+
+    - name: Add Felix Krull's Python PPA
+      apt_repository:
+        repo: 'ppa:fkrull/deadsnakes'
+        state: present
+
     - name: Update apt cache
       apt: update-cache=yes cache_valid_time=86400
       become: yes
+      become_method: sudo
 
-    - name: Check NTP service to make sure its running
-      service: name=ntp state=started enabled=yes
-      become: yes
+    - name: Install Python 3.6
+      apt:
+        name: python3.6
+        state: latest
 
-    - name: Install AWS CLI package via Pip
-      pip: name=awscli
+    - name: install python-pip
+      apt: name=python-pip
       become: yes
 
-    - name: Install Boto Python package via Pip
-      pip: name=boto
+    - name: install python-virtualenv
+      apt: name=python-virtualenv
       become: yes
+
+    - name: Install PIP requirements
+      pip:
+        requirements: /vagrant/ansible/pip-requirements.txt
+        virtualenv: /home/vagrant/python3.6
+        virtualenv_python: python3.6
+
+#    - name: Manually create the initial virtualenv
```

Comment:
The commented out lines? No, I had left them in there as a reference, but you're right they're not needed, i'll nuke them.